code/core/src/manager/settings/Checklist/Checklist.tsx

```diff
@@ -0,0 +1,410 @@
+    borderRadius: 8,
+  },
+}));
+
+const Items = styled.ol(({ theme }) => ({
+  listStyle: 'none',
+  display: 'flex',
+  flexDirection: 'column',
+  margin: 0,
+  padding: 0,
+
+  '& > li:not(:last-child)': {
+    boxShadow: `inset 0 -1px 0 ${theme.color.border}`,
+  },
+
+  '& > li:last-child': {
+    borderBottomLeftRadius: 7,
+    borderBottomRightRadius: 7,
+  },
+}));
+
+const SectionSummary = styled.div<{ progress: number; isCollapsed: boolean }>(
+  ({ theme, progress, isCollapsed }) => ({
+    position: 'relative',
+    fontWeight: 'bold',
+    display: 'flex',
+    alignItems: 'center',
+    gap: 10,
+    padding: '10px 10px 10px 15px',
+    borderBottom: `5px solid ${theme.base === 'dark' ? theme.color.darker : theme.color.light}`,
+    borderBottomLeftRadius: isCollapsed ? 7 : 0,
+    borderBottomRightRadius: isCollapsed ? 7 : 0,
+    transition: 'border-radius var(--transition-duration, 0.2s)',
+    '--toggle-button-rotate': isCollapsed ? '0deg' : '180deg',
+    '--toggle-button-opacity': 0,
+
+    '&:hover, &:focus-visible': {
+      outline: 'none',
+      '--toggle-button-opacity': 1,
+    },
+
+    h3: {
+      flex: 1,
+      margin: 0,
+      fontSize: 'inherit',
+      fontWeight: theme.typography.weight.bold,
+    },
+
+    '&::after': {
+      pointerEvents: 'none',
+      position: 'absolute',
+      top: 0,
+      bottom: -5,
+      left: 0,
+      right: 0,
+      content: '""',
+      display: 'block',
+      width: `${progress}%`,
+      borderBottom: `5px solid ${theme.color.positive}`,
+      borderBottomLeftRadius: 'inherit',
+      borderBottomRightRadius: progress === 100 ? 'inherit' : 0,
+      transition: 'width var(--transition-duration, 0.2s)',
+    },
+  })
+);
+
+const SectionHeading = styled.h3({
+  cursor: 'default',
+});
+
+const ItemSummary = styled.div<{ isCollapsed: boolean }>(({ isCollapsed }) => ({
+  fontWeight: 'normal',
+  display: 'flex',
+  alignItems: 'center',
+  gap: 10,
+  padding: isCollapsed ? '6px 10px 6px 15px' : '10px 10px 10px 15px',
+  transition: 'padding var(--transition-duration, 0.2s)',
+  '--toggle-button-rotate': isCollapsed ? '0deg' : '180deg',
+
+  '&:focus-visible': {
+    outline: 'none',
+  },
+
+  h4: {
+    flex: 1,
+    margin: 0,
+    fontSize: 'inherit',
+  },
+}));
+
+const ItemHeading = styled.h4<{ skipped: boolean }>(({ theme, skipped }) => ({
+  color: skipped ? theme.color.mediumdark : theme.color.defaultText,
+  cursor: 'default',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+}));
+
+const ItemContent = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  gap: 8,
+  padding: '0 15px 15px 15px',
+
+  p: {
+    margin: 0,
+    lineHeight: 1.4,
+  },
+});
+
+const StatusIcon = styled.div(({ theme }) => ({
+  position: 'relative',
+  flex: '0 0 auto',
+  minHeight: 16,
+  minWidth: 16,
+  margin: 0,
+  background: theme.background.app,
+  borderRadius: 9,
```

Comment:
General question: Do we not have a good set of theme variables that define e.g. different font sizes or border radius? We use a lot of hardcoding of styles, and I am curious whether we can reuse some style variables/theme variables instead. cc @MichaelArestad